Protecs/src/test/java/uk/ac/ucl/protecs/helperFunctions/helperFunctions.java

```diff
@@ -0,0 +1,33 @@
+package uk.ac.ucl.protecs.helperFunctions;
+
+import java.util.ArrayList;
+import java.util.Random;
+import uk.ac.ucl.protecs.sim.Params;
+import uk.ac.ucl.protecs.sim.WorldBankCovid19Sim;
+
+public class helperFunctions {
+	
+	public static WorldBankCovid19Sim CreateDummySimWithRandomSeed(String paramsFilename, boolean demography) {
```

Comment:
we could overload this constructor! That is, if they call this, just call the other constructor with a random integer in the params

Reply:
Switched both to be called CreateDummySim!